js/src/analysis/isSEMrushIntegrationActive.js

```diff
@@ -0,0 +1,14 @@
+import getL10nObject from "./getL10nObject";
+
+import { get } from "lodash-es";
```

Comment:
We're being largely inconsistent through the codebase but it's a good practice to add comments to split External dependencies, Internal dependencies, etc, in groups.

Reply:
Yeah, my fault